src/cnumber.jl

```diff
@@ -26,9 +26,8 @@ Base.promote_rule(::Type{<:CNumber},::Type{<:Number}) = CNumber
 Base.one(::Type{Parameter}) = 1
 Base.zero(::Type{Parameter}) = 0
 Base.adjoint(x::SymbolicUtils.Symbolic{<:CNumber}) = conj(x)
+Base.adjoint(x::SymbolicUtils.BasicSymbolic{Complex{Real}}) = _conj(x)
```

Comment:
That's type piracy. We neither own the type `BasicSymbolic{Complex{Real}}` nor the function `adjoint`. This should be defined in SymbolicUtils.jl. We should raise an issue with them. I think they'll be fine with adding the methods since that's how numbers behave in Julia.